utils/appliance/__init__.py

```diff
@@ -1633,7 +1633,7 @@ def wait_for_evm_service(self, timeout=900, log_callback=None):
         return result
 
     @logger_wrap("Rebooting Appliance: {}")
-    def reboot(self, wait_for_web_ui=True, log_callback=None):
+    def reboot(self, wait_for_web_ui=True, quit_browser=False, log_callback=None):
```

Comment:
**REQUIRED**: We also discussed to move the reboot into the fixture and remove this prototype